tools/system_libs.py

```diff
@@ -760,7 +760,9 @@ def __init__(self, **kwargs):
 
   def get_cflags(self):
     cflags = super().get_cflags()
-    if not self.is_debug:
+    if self.is_debug:
+      cflags += ['-fstack-protector']
```

Comment:
do we really want to turn fstack-protector on for all debug libraries?
If so, this is a more user-visible change than just making the option work, so we should mention it in the changelog.
Also are there any other emscripten docs that should be updated now that -fstack-protector works?

Reply:
I'm happy to revert this part... although I don't see it being very impactfull.   The majority of functions are not effected by `-fstack-protector` (see https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-fstack-protector).

Reply:
Reverted